changelog/upgrade-notes.md

```diff
@@ -6,6 +6,11 @@
 
 The experimental `setHue`, `setContrast`, `setBrightness`, `setSaturation`, and the corresponding getter methods have been removed.  These properties only worked with the WebGL renderer.  If are interested in applying color transforms, look for the `postcompose` event in the API docs.  In addition, the `ol.source.Raster` source provides a way to create new raster data based on arbitrary transforms run on any number of input sources.
 
+#### Removal of `ol.ENABLE_NAMED_COLORS`
+
+This option existed to support the use of Google Closure library named colors and was removed as a part of OpenLayers moving away from Google Closure library. Note that this do not affect the use of CSS named colors.
```

Comment:
Should we more explicit here? Sth. like: if you have used the canvas renderer (which is the default) for your vector layers, this change will not affect you. If you have been using the webgl renderer though, you cannot any longer use named colors but must provide them in <a suitable format.